File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        String change = myString.replace("A","temp") .replace("B","A") .replace("temp","B");
        if(change.contains(pat)){
            answer = 1;
        }
        else{ answer = 0;
        }
        return answer;
    }
}

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.js
function solution(myString, pat) {
    var answer = '';
    for(let i=0; i<myString.length; i++){
        if(myString.charAt(i)==="A") answer+="B";
        else if(myString.charAt(i)==="B") answer +="A";
    }
    return answer.includes(pat)?1:0;
}